source/game/field/ObjectFlowTable.hh

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Common.hh>
+
+namespace Field {
+
+struct SObjectCollisionSet {
+    u16 id;
+    char name[32];
+    char resources[64];
+    u8 _62[0x6a - 0x62];
+    s16 mode;
+    union {
+        struct {
+            s16 radius;
+        } sphere;
+        struct {
+            s16 radius;
+            s16 height;
+        } cylinder;
+        struct {
+            s16 x;
+            s16 y;
+            s16 z;
+        } box;
+    } params;
+    u8 _72[0x74 - 0x72];
+};
+static_assert(sizeof(SObjectCollisionSet) == 0x74);
+
+class ObjectFlowTable {
+public:
+    ObjectFlowTable(const char *filename);
+    ~ObjectFlowTable();
+
+private:
+    struct File {
```

Comment:
I think this should be renamed to avoid ambiguity with C File pointers. I know it's scoped within `ObjectFlowTable`, but if it's used in the same TU then it may look ambiguous.

Reply:
Renamed to `SFile`.